### DISCUSSION TITLE
Change the guard

### DISCUSSION THREAD

**Discussion by msucevan** [MohmmedAshraf/laravel-translations#76] [Q&A]
Is possible to change the guard, I'm using laravel nova and I want to have the same login. Thanks

**Comment by ChrisToxz** [comment]
@MohmmedAshraf any thoughts about this?
I'm not using Nova but also I would like to have just a single login and user database.
I understand that the contributors contains a role, but that could be stored in a different way too.

In my case users are logging in through Azure, so its a deal breaker to have an additional account, but would love to use this package..
